widgets/ShareLink.php

```diff
@@ -24,10 +24,6 @@ class ShareLink extends \yii\base\Widget
      */
     public function run()
     {
-        if (Yii::$app->controller instanceof ContentContainerController) {
-            return;
-        }
```

Comment:
not sure why this was added. It prevents the share button to show up in all other places than dashboard, why is that?